_data/ecosys-submissions.csv

```diff
@@ -0,0 +1,4 @@
+id,product_category,product_name,product_link,download_link,brief_description
+2,Producer,test,testingg,,teaowjei aiwjeg oiajweoj gtaow eia weijgoja wiegjawe o
```

Comment:
Is this file required? I see some items may have some userids which may have changed for some users.
```suggestion
2,Producer,test,testing,,teaowjei aiwjeg oiajweoj gtaow eia weijgoja wiegjawe o
```

Reply:
Good catch! This file is used as a sample to demonstrate pulling data from a CSV file to reference when creating a solution for storing/retrieving data, but I should comment out where the file is being used for now.

Reply:
The "typo" of "testingg" is how the data was submitted to the form, so I'll leave that part as is in case people want to reference the raw form submissions at any point.